docs/backend/quantization.ipynb

```diff
@@ -0,0 +1,281 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Quantization\n",
+    "\n",
+    "`SGLang` support various quantization methods, including online dynamic quantization and offline quantization.\n",
```

Comment:
State it here that we do not recommend online quantization (not for torchao), that's only for the acitivation.